ui/pages/desktop-pairing/index.scss

```diff
@@ -3,126 +3,39 @@
   flex-flow: column;
   align-items: center;
   padding: 0 30px 0;
+  max-width: 640px;
 
   &__countdown-timer {
-    background: #f2f3f4;
-    border-radius: 15.5px;
-    padding: 7px 0 7px 0;
-    margin: 0 32px 0 32px;
+    background: var(--color-background-default);
+    border-radius: 16px;
+    padding: 8px 0;
     font-size: 12px;
+    width: 240px;
```

Comment:
Does this need a width? Random static widths are sometimes are red flag for me

Reply:
I've removed this, but still need to define some width in order to prevent flickering of this component.
<img width="283" alt="Screenshot 2023-03-21 at 12 16 22" src="https://user-images.githubusercontent.com/7644512/226590461-b3fbc614-f74e-4be2-963d-b9a00a212138.png">
I've defined min-width for that, I think makes much more sense than setting width